src/styles/global.ts

```diff
@@ -182,6 +183,681 @@ const GlobalStyles = createGlobalStyle`
   .noScroll {
     overflow: hidden;
   }
+
+  /* Toastify custom styling */
+  .Toastify__toast-container {
```

Comment:
As this quite a big addition, I would prefer all `/* Toastify custom styling */` in its own var in a separate file
```
 const GlobalStyles = createGlobalStyle`
    ...
    ${ToastifyStyles}
    ...
`
```